tests/cli/test_gen_assembly.py

```diff
@@ -0,0 +1,177 @@
+from unittest import TestCase
+from unittest.mock import MagicMock, Mock, patch
+
+from flexmock import flexmock
+
+import doozerlib
+from doozerlib.assembly import AssemblyIssueCode, AssemblyTypes, AssemblyIssue
+from doozerlib.cli.release_gen_assembly import GenAssemblyCli
+
+
+class TestGenPayloadCli(TestCase):
```

Comment:
Really love the test coverage, and how clean it looks!

Could you maybe add a docstring to the `test_` methods explaining what business logic this is covering? I noticed that method name of the test and the input and the assertion did not always make it clear for me what this is really about.